### PR TITLE
{lib,math}[GCCcore/12.3.0,gfbf/2023a] Transformers v4.39.3, tokenizers v0.15.2

### PR DESCRIPTION
**Pull request by VRehnberg**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/t/tokenizers/tokenizers-0.15.2-GCCcore-12.3.0-Rust-1.75.0.eb [hunk 1]
```diff
@@ -0,0 +1,329 @@
+easyblock = 'CargoPythonBundle'
+
+name = 'tokenizers'
+version = '0.15.2'
+_rust_ver = '1.75.0'
+versionsuffix = '-Rust-%s' % _rust_ver
```
Comment (Micket) [inline easybuild/easyconfigs/t/tokenizers/tokenizers-0.15.2-GCCcore-12.3.0-Rust-1.75.0.eb:6]: Don't need a suffix for a builddep only.

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by VRehnberg** [comment]
Test report by @VRehnberg
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis1-13 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6226R CPU @ 2.90GHz, 1 x NVIDIA Tesla V100-SXM2-32GB, 550.54.14, Python 3.6.8
See https://gist.github.com/VRehnberg/c3b2c1f6cdfad2607a901ddb46f33be1 for a full test report.

**Comment by VRehnberg** [comment]
Test report by @VRehnberg
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
alvis1-13 - Linux Rocky Linux 8.9, x86_64, Intel(R) Xeon(R) Gold 6226R CPU @ 2.90GHz, 1 x NVIDIA Tesla V100-SXM2-32GB, 550.54.14, Python 3.6.8
See https://gist.github.com/VRehnberg/69325bb64e7e119781f902a2d413da25 for a full test report.

**Comment by VRehnberg** [comment]
Test report by @VRehnberg
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
alvis-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/VRehnberg/54edf8c95f421f24c675e8818e08625d for a full test report.

**Comment by laraPPr** [comment]
Test report by @laraPPr
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node4006.donphan.os - Linux RHEL 8.8 (Ootpa), x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz, 1 x NVIDIA NVIDIA A2, 545.23.08, Python 3.11.3
See https://gist.github.com/laraPPr/ca55ad2b3e7b05a7cb1b56a394210ce4 for a full test report.

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3101.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/ce75681eb067783c74461ced9fe74085 for a full test report.

**Comment by laraPPr** [comment]
@boegelbot please test @ jsc-zen3

**Comment by boegel** [comment]
Going in, thanks @VRehnberg!

**Pull request merged**